dasgo/requests.go

```diff
@@ -1077,6 +1077,8 @@ type GetGuildWidget struct {
 	GuildID Snowflake
 }
 
+// TODO: no table
```

Comment:
The GuildID comes from the request parameter `{guild.id}`. The other fields are speculation on the provided JSON:
```json
{
  "code": "abc",
  "uses": 12
}
```

Otherwise, a partial object would refer to simply using the object _(i.e `Invite`)_. It seems that is not the case here so we use the provided JSON.

Reply:
should I leave as optional?

Reply:
Yes.